src/studio/src/designer/backend.Tests/.gitignore

```diff
@@ -0,0 +1 @@
+/_TestData/Repositories
```

Comment:
Won't this ignore the /_TestData/Repositories folder even for test repositories that should be added as part of the tests?

Reply:
I guess so, but as far as I could see, no such folder has been added to git yet, but files were generated there when I ran the test suite. Maybe they're supposed to be deleted automatically after, but a lot of the unit tests are failing on my machine right now - I guess they haven't been tested thoroughly on Linux.

Reply:
@tjololo did a lot of changes to get the tests running on mac. I was hoping those changes would be enough to have the tests run on linux as well. The primary issue was the use of backslash (\) in paths, which he changed to slash (/).

Reply:
Yep, I asked him about the failing tests in the designer backend, but they seemed to be deeper issues than just backslashes in paths. The relevant tests for this issue was working well enough. If the dangling generated files in `_TestData/Repositories` are just a side-effect of the failing tests on my computer, I can remove this gitignore file.

Reply:
Hm, it might be that at one point _TestData was intended for temp testdata files, but it looks like there is an expectation that it should exist and that the dangling files are just a side effect. We should write temp files to an isolated folder with a .gitignore like you suggest, but I think they are written to the same folder. In some tests we have a try/finally which cleans up but it's not used everywhere.

Reply:
Yep, to me it looked like there was generated/temp files only in the `_TestData/Repositories` folder. I don't see the need to restructure this, as I guess the generated files in that folder qualifies for 'test data' as well, even if they're (probably) supposed to be deleted after a test run. This change to `.gitignore` should only ignore the generated/temp files though, not the ones already added to git.

Reply:
But if we need to add more test repositories under testuser/ttd then those won't be added either?

Reply:
Sorry, I'm not sure if I fully understand the question. _Yes_, it would be a problem if you wanted to add some files under this location to git (not something anyone has done yet?), but _no_, it wouldn't be a problem when running the tests, or if new test cases are added that generate even more files in the directory. Files will be generated and stored locally as before, they are just ignored by git. The reason I added this was because I didn't want to have all these dangling generated files showing up as noise when running `git status`.

Reply:
For posterity: My mistake! I had somehow concluded there was no files committed to the Repositories folder, but I was obviously mistaken. Removing the .gitignore. Now that tests are running properly on Linux, these files are also removed properly, so there is not much need for the .gitignore.